src/meson.build

```diff
@@ -48,4 +48,18 @@ libglyphy = library('glyphy', glyphy_sources + glyphy_headers + glyphy_shader_so
   version: '0.0.0',
   install: true)
 
+libglyphy_dep = declare_dependency(
+  link_with: libglyphy,
+  include_directories: [confinc],
+  sources: glyphy_sources + glyphy_headers + glyphy_shader_sources)
+
+meson.override_dependency('glyphy', libglyphy_dep)
+
+libglyphy_pc = pkgmod.generate(libglyphy,
+  name: 'glyphy',
+  description: 'High-quality glyph rendering library using OpenGL ES2 shaders',
+  version: meson.project_version(),
+  subdirs: meson.project_name(),
+  install_dir: join_paths(libdir, 'pkgconfig'))
```

Comment:
This install_dir is the default so you don't need to override it.

The version defaults to project_version() if you don't specify it.

The name defaults to the same name as the library name of the library specified in the optional first positional argument, which you have. So in this case it should be unneeded.

Reply:
Cool, will update too